Dockerfile

```diff
@@ -2,8 +2,8 @@ FROM node:5
 
 RUN apt-get install libpq-dev
 
-COPY package.json /
+WORKDIR /sequelize
+COPY package.json /sequelize/
 RUN npm install
 
-WORKDIR /sequelize
-VOLUME /sequelize
+COPY . /sequelize/
```

Comment:
please don't edit dev environment files when you know for a fact that a lot of other people rely on them :)